### PR TITLE
Added schemas for weather configuration

### PR DESCRIPTION
**Pull request by AALEKH**

### CODE REVIEW

**Comment by alexanderdean** [comment]
Hey @AALEKH! It's coming along. I think we will need a field for the Wunderground API key too...

**Comment by AALEKH** [comment]
Cool, Will include it

File: schemas/com.snowplowanalytics.snowplow/weather_config/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		},
+		"parameters": {
+			"apiKey": {
+				"type": "string"
+			},
+			"required": ["apiKey"],
+			"additionalProperties": false
+		}
+	},
+	"required": ["name", "vendor", "enabled"],
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/weather_config/jsonschema/1-0-0]: I think "parameters" has to be in this array ^^ too...

Comment (AALEKH) [inline schemas/com.snowplowanalytics.snowplow/weather_config/jsonschema/1-0-0]: Ok, will change this accordingly

**Comment by alexanderdean** [comment]
Also @AALEKH : rename com.snowplowanalytics.snowplow/weather_information to com.wunderground/weather

File: schemas/com.snowplowanalytics.snowplow/weather_information/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+        "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+        "description": "Schema for information of weather enrichment",
+        "self": {
+                "vendor": "com.snowplowanalytics.snowplow",
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/weather_information/jsonschema/1-0-0]: Let's change this to org.openweathermap/weather/jsonschema/1-0-0

**Comment by AALEKH** [comment]
cc: @alexanderdean

**Comment by alexanderdean** [comment]
Gosh sorry @AALEKH - my comment was meant for @chuwy!

**Comment by chuwy** [comment]
Why wunderground?

**Comment by alexanderdean** [comment]
Sorry for confusion, this was a brain freeze yesterday, I was looking at these PRs as if they were created by @chuwy not @AALEKH . Closing to prevent further confusion.

**Pull request closed without merging**